### DISCUSSION TITLE
Aux POW missing chain merkle root in parent coinbase

### DISCUSSION THREAD

**Discussion by levon1991** [dogecoin/dogecoin#2699] [Q&A]
Hi,

I've installed latest wallet but have a some problem with merged mining. 
I try to merge with LTC and can find a block but when I try to submit it on Doge node it return

2021-11-12 12:20:02 ERROR: Aux POW missing chain merkle root in parent coinbase
2021-11-12 12:20:02 ERROR: CheckAuxPowProofOfWork : AUX POW is not valid
2021-11-12 12:20:02 ERROR: ProcessNewBlock: AcceptBlock FAILED

can you help me to main LTC+DOGE not only LTC?))

**Comment by patricklodder** [comment]
Did it work on the previous version or is this a new installation?

**Reply by levon1991** [reply]
I also want to clarify that the tests are carried out on Testnet (LTC, DOGE)

**Comment by levon1991** [comment]
this is a new installation

**Comment by levon1991** [comment]
To be honest, the problem may be on our side, because we are just starting to implement merged mining, now we have 2751 Gh / s, but we only use it for LTC and want to merged with DOGE

**Reply by dreams-money** [reply]
Did you find what the issue was? - I'm experiencing the same message.
